service/create/service.go

```diff
@@ -374,11 +372,42 @@ func (s *Service) Boot() {
 					}
 					s.logger.Log("info", "deleted workers")
 
+					// Delete S3 bucket objects
+					var bucket resources.Resource
+					bucket = &awsresources.Bucket{
+						Name:      awsresources.BucketName(cluster),
+						AWSEntity: awsresources.AWSEntity{Clients: clients},
+					}
+
+					var masterBucketObject resources.Resource
+					masterBucketObject = &awsresources.BucketObject{
+						Name:      awsresources.BucketObjectName(cluster, prefixMaster),
+						Bucket:    bucket.(*awsresources.Bucket),
+						AWSEntity: awsresources.AWSEntity{Clients: clients},
+					}
+					if err := masterBucketObject.Delete(); err != nil {
+						s.logger.Log("error", errgo.Details(err))
```

Comment:
MaskAny is for masking the error which we want to return.

errgo.Details is for displaying the info in the stringified format, with the full "traceback" we provided by masking that error before.

Here we log an error, not return it.